packages/service-clients/azure-client/api-report/azure-client.api.md

```diff
@@ -68,7 +68,7 @@ export interface AzureContainerVersion {
 
 // @internal @deprecated
 export class AzureFunctionTokenProvider implements ITokenProvider {
-    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "userId" | "userName" | "additionalDetails"> | undefined);
```

Comment:
CI seems to be expecting this change, so I un-reverted its addition. Unclear what changed API-Extractor's behavior, but this change seems to be necessary to satisfy CI.